azure_translator/__init__.py

```diff
@@ -72,7 +72,7 @@ def translate(self, text, to=DEFAULT_LANGUAGE, source_language=None):
             raise self._api_error(error)
 
         try:
-            return ET.fromstring(resp.content.encode('utf-8')).text
+            return ET.fromstring(resp.content).text
```

Comment:
from https://github.com/kennethreitz/requests/blob/master/requests/models.py#L784-L840 : `request.content` is not decoded, `request.text` is.